src/Servers/Kestrel/Core/src/BadHttpRequestException.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
I wish git better tools around normalizing BOMs similar to what it has for line endings gitattributes.